cookbooks/bcpc-hadoop/recipes/zookeeper_packages.rb

```diff
@@ -27,9 +27,17 @@
 include_recipe 'bcpc-hadoop::hdp_repo'
 
 package hwx_pkg_str('zookeeper-server',
-                    node[:bcpc][:hadoop][:distribution][:release]) do
+                    node['bcpc']['hadoop']['distribution']['release']) do
+  action :upgrade
+end
+
+package hwx_pkg_str('zookeeper-client',
```

Comment:
I don't see us installing a `zookeeper-client` package; is this necessary or do we only need to change the symlink (i.e. run `hdp-select`)?

Reply:
good catch.

Reply:
`zookeeper-client` is just a symlink generated by hdp-select pointing to the actual zookeeper-package.

The `zookeeper-server` package seems to have a `hdp-select zookeeper-client` invoked in the post-install script that doesnt' get executed in package upgrades.